plugins/modules/cloud/misc/proxmox_template.py

```diff
@@ -49,7 +49,6 @@
     description:
       - Proxmox VE node, when you will operate with template
     type: str
-    required: true
```

Comment:
This change seems wrong, without the `node` parameter it is not possible to reach the necessary API calls.

Reply:
In that case it's not the docs that are wrong, but the code. The argument spec (implicitly) says that this is not required: https://github.com/russoz/community.general/blob/27a11715129311dc11fed1283e755e462f1c5bcc/plugins/modules/cloud/misc/proxmox_template.py#L213

Reply:
Ah yes, I can see that. One more thing on my TODO list then.